### PR TITLE
[HT] Test if all the needed material properties read from project file.

### PR DESCRIPTION
**Pull request by TomFischer**
Until now, OGS crashes without a useful error message when a material property isn't specified in the project file.

1. [x] Feature description was added to the [changelog](https://github.com/ufz/ogs/wiki/Release-notes-6.2.2)
2. [x] Tests covering your feature were added? (most of the added lines are tests)

### CODE REVIEW

**Comment by TomFischer** [comment]
@bilke I haven't any idea why netlify is failing. Can you have a look, please?

File: ProcessLib/HT/Tests.cmake [hunk 1]
```diff
@@ -448,3 +448,24 @@ AddTest(
...
+
+if(NOT OGS_USE_MPI AND BUILD_TESTING AND Python_FOUND)
+    add_custom_target(generate_invalid_project_files_ht
+        ${Python_EXECUTABLE}
+        ${PROJECT_SOURCE_DIR}/ThirdParty/ogs6py/generateInvalidMediaForHT.py
+                                                generateInvalidMediaForHT.py
+        WORKING_DIRECTORY ${Data_SOURCE_DIR}/Parabolic/HT/InvalidProjectFiles/)
+    file(GLOB HT_INVALID_PRJ_FILES ${Data_SOURCE_DIR}/Parabolic/HT/InvalidProjectFiles/*.prj)
+    foreach(ht_invalid_prj_file ${HT_INVALID_PRJ_FILES})
+        string(REPLACE ${Data_SOURCE_DIR}/Parabolic/HT/InvalidProjectFiles/HT "invalid" ht_invalid_prj_file_short ${ht_invalid_prj_file})
```
Comment (endJunction) [inline ProcessLib/HT/Tests.cmake]: Shouldn't it be the binary dir? Polluting source dir is not nice ;)

File: MaterialLib/MPL/Medium.cpp [hunk 2]
```diff
@@ -47,6 +47,11 @@ Property const& Medium::property(PropertyType const& p) const
     return *_properties[p];
 }
 
+bool Medium::isProperty(PropertyType const& p) const
```
Comment (endJunction) [inline MaterialLib/MPL/Medium.cpp]: It's maybe more the question *has* property, isn't it?

Comment (TomFischer) [reply MaterialLib/MPL/Medium.cpp]: Yes, maybe `hasProperty` sounds more appropriate. Will change this.

File: ProcessLib/HT/Tests.cmake (hunk 1 above)
Comment (TomFischer) [reply ProcessLib/HT/Tests.cmake]: Will change this tomorrow. Please don't merge.

**Comment by bilke** [comment]
> @bilke I haven't any idea why netlify is failing. Can you have a look, please?

```
Fetched in submodule path 'ThirdParty/ogs6py', but it did not contain 51afd2cec3678d66a00dd2e8436c9ba1d3335e3b. Direct fetching of that commit failed.
```

**Pull request merged**